modules/m_away.c

```diff
@@ -89,6 +90,10 @@ m_away(struct MsgBuf *msgbuf_p, struct Client *client_p, struct Client *source_p
 
 			sendto_common_channels_local_butone(source_p, CLICAP_AWAY_NOTIFY, NOCAPS, ":%s!%s@%s AWAY",
 							    source_p->name, source_p->username, source_p->host);
+			struct monitor *monptr = find_monitor(source_p->name, 0);
+			if(monptr)
+				sendto_monitor_with_capability_butserial(source_p, monptr, CLICAP_EXTENDED_MONITOR | CLICAP_AWAY_NOTIFY, NOCAPS, true, ":%s!%s@%s AWAY",
```

Comment:
Eliminate current_serial abuse

```suggestion
				sendto_monitor_with_capability(source_p, monptr, CLICAP_EXTENDED_MONITOR | CLICAP_AWAY_NOTIFY, NOCAPS, ":%s!%s@%s AWAY",
```